.github/workflows/deploy.yml

```diff
@@ -31,5 +31,5 @@ jobs:
             git checkout FETCH_HEAD
             killall -9 vitess-bot
             source ~/.profile
-            go build -o vitess-bot ./...
+            go build -o vitess-bot ./go
```

Comment:
this is because we now have subpackages; it won't build without it because now it's trying to build multiple packages (the subtree) into one binary